packages/components/form/src/form-item.vue

```diff
@@ -159,11 +160,22 @@ const hasLabel = computed<boolean>(() => {
 })
 
 const labelFor = computed<string | undefined>(() => {
-  return props.for || inputIds.value.length === 1
+  return isShouldAssociated.value && (props.for || inputIds.value.length === 1)
     ? inputIds.value[0]
     : undefined
 })
 
+const associatedElement = () => {
+  const id = inputIds.value[0]
+  const el = formItemRef.value?.querySelector(`#${id}`)
+  if (el) {
+    const tag = el.tagName.toLowerCase()
+    if (tag !== 'div') {
+      isShouldAssociated.value = true
+    }
+  }
+}
+
```

Comment:
IMO, The modifications to this file are unnecessary. Because when deleting the `useFormItemInputId` from checkbox-group and radio-group, the value of the `labelFor` becomes undefined. Also avoid modifying these test files